fontir/src/ir.rs

```diff
@@ -63,6 +69,161 @@ impl StaticMetadata {
     }
 }
 
+/// See <https://learn.microsoft.com/en-us/typography/opentype/spec/name#name-ids>
+#[derive(Serialize, Deserialize, Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub enum NameId {
```

Comment:
this should maybe be part of `write-fonts`?

Reply:
Does write-fonts do serde?

Reply:
(that's the only reason why not that comes to mind and even that seems minor)

Reply:
> lgtm modulo cosimo's approval

I'm going to merge and then fix all the problems @anthrotype points out in a follow-on so I can establish a clean base for .glyphs support.